test/Common/HttpApiHandlerTests.cs

```diff
@@ -919,65 +921,68 @@ public async Task GetEntity_Returns_State_Or_HTTP_404(bool hasKey, bool exists)
         [Trait("Category", PlatformSpecificHelpers.TestCategory)]
         public async Task Entities_Query_Calls_ListEntitiesAsync(bool hasName)
         {
-            string entityName = hasName ? Guid.NewGuid().ToString("N") : "";
-            var uriBuilder = new UriBuilder(TestConstants.NotificationUrl);
-
-            uriBuilder.Path += $"/entities/{entityName}";
+            // Build mock
+            string entityName = Guid.NewGuid().ToString("N");
 
-            var testRequest = new HttpRequestMessage
-            {
-                Method = HttpMethod.Get,
-                RequestUri = uriBuilder.Uri,
-            };
-
-            var list = (IReadOnlyCollection<DurableEntityStatus>)new List<DurableEntityStatus>
+            var mockList = (IReadOnlyCollection<DurableEntityStatus>)new List<DurableEntityStatus>
             {
                 new DurableEntityStatus
                 {
-                    EntityId = new EntityId("DoThis", "one"),
+                    EntityId = new EntityId(entityName, "one"),
                     LastOperationTime = new DateTime(2018, 3, 10, 10, 10, 10, DateTimeKind.Utc),
                     State = null,
                 },
                 new DurableEntityStatus
                 {
-                    EntityId = new EntityId("DoThat", "two"),
+                    EntityId = new EntityId(entityName, "two"),
                     LastOperationTime = new DateTime(2018, 3, 10, 10, 6, 10, DateTimeKind.Utc),
                     State = null,
                 },
             };
 
-            var result = new EntityQueryResult() { Entities = list, ContinuationToken = null };
+            var mockResult = new EntityQueryResult() { Entities = mockList, ContinuationToken = null };
             var clientMock = new Mock<IDurableClient>(MockBehavior.Strict);
 
             clientMock
                     .Setup(x => x.ListEntitiesAsync(It.IsAny<EntityQuery>(), It.IsAny<CancellationToken>()))
-                    .Returns(Task.FromResult(result));
+                    .Returns(Task.FromResult(mockResult));
 
-            var httpApiHandler = new ExtendedHttpApiHandler(clientMock.Object);
+            // Build Uri
+            var uriBuilder = new UriBuilder(TestConstants.NotificationUrl);
+
+            if (hasName)

```

Comment:
This is much better! Can we rename this from `hasName` to `useNameFilter`? That will make it clear that this is not about whether an entity has a name (they both do), but rather it's about whether we're using the entity name filter.